From bbec72e4382b3167b0c002219ee239b4d0d29782 Mon Sep 17 00:00:00 2001
From: Joseph Szczypek <jszczype@redhat.com>
Date: Mon, 26 Jan 2015 21:43:49 -0500
Subject: [scsi] hpsa: add in P840ar controller model name

Message-id: <1422308629-3927-3-git-send-email-jszczype@redhat.com>
Patchwork-id: 103575
O-Subject: [RHEL7.1 e-stor PATCH 2/2] hpsa: add in P840ar controller model name
Bugzilla: 1185467
RH-Acked-by: Tomas Henzl <thenzl@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

Add upstream change to bring in P840ar gen9 model name:
www.spinics.net/lists/linux-scsi/msg82108.html

    hpsa: add in P840ar controller model name

    Add in P840ar model name for gen9

    Reviewed-by: Scott Teel <scott.teel@pmcs.com>
    Signed-off-by: Don Brace <don.brace@pmcs.com>

Change was manually added.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index a9ee35f..778075b 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -173,7 +173,7 @@ static struct board_type products[] = {
  {0x21BE103C, "Smart Array P741m", &SA5_access},
  {0x21BF103C, "Smart HBA H240ar", &SA5_access},
  {0x21C0103C, "Smart Array P440ar", &SA5_access},
- {0x21C1103C, "Smart Array", &SA5_access},
+ {0x21C1103C, "Smart Array P840ar", &SA5_access},
  {0x21C2103C, "Smart Array P440", &SA5_access},
  {0x21C3103C, "Smart Array P441", &SA5_access},
  {0x21C4103C, "Smart Array", &SA5_access},
-- 
1.7.1